src/scenario_tests.rs

```diff
@@ -183,6 +184,7 @@ fn run_test(
             action_our_events: action.our_events,
             action_our_section: action.our_section,
             action_merge_infos: action.merge_infos,
+            action_merge_needed: action.merge_needed,
```

Comment:
This should not be here, we set it ourselves so we don't want to check its value.

Reply:
Good catch! I'll remove it and squash